controller/config.go

```diff
@@ -112,7 +112,12 @@ const (
 	// DefaultMaxTxnLogCollectionMB is the maximum size the txn log collection.
 	DefaultMaxTxnLogCollectionMB = 10 // 10 MB
 
-	JujuHASpace         = "juju-ha-space"
+	// JujuHASpace is the network space within which the MongoDB replica-set
+	// should communicate.
+	JujuHASpace = "juju-ha-space"
+
+	// JujuManagementSpace is the network space within which controllers in a HA
+	// set-up should communicate.
```

Comment:
It is the space that agents should use to communicate with controllers.

Or the space where controllers should advertise their addresses to agents?